src/sdk/strategy/cbd-strategy.ts

```diff
@@ -30,15 +30,13 @@ export class CbdStrategy {
   }
 
   public async deploy(
-    web3Provider: ethers.providers.Web3Provider
+    web3Provider: ethers.providers.Web3Provider,
+    blockUntilDeployed = true
```

Comment:
Is this a sensible default?

Reply:
It doesn't sound bad to me, but maybe I'm not the best person to say it.

Reply:
i like it, you don't want to get in the situation of attempting to use a ritual that never successfully completed